main/test/ca/uwaterloo/flix/language/phase/TestTyper.scala

```diff
@@ -1916,21 +1916,4 @@ class TestTyper extends FunSuite with TestUtils {
         |""".stripMargin
     expectError[TypeError.GeneralizationError](compile(input, Options.TestWithLibNix))
   }
-
-  test("TestCaseSetAnnotation.05") {
```

Comment:
This is a negative test that was removed? Should we be scared?

Reply:
Its much harder to observe this issue now since constructors "add on top"